app-benchmarks/stress-ng/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME=stress-ng
+PKGSEC=utils
+PKGDEP="apparmor gmp kmod libbsd libglvnd libjpeg-turbo lksctp-tools mpfr xxhash"
+PKGDEP__AMD64="${PKGDEP} intel-ipsec-mb"
+BUILDDEP="attr eigen-3 keyutils libaio libcap libgcrypt libglvnd libmd"
+PKGDES="Software to stress test a computer system in various selectable ways"
+
+ABTYPE=plainmake
+
+NOLTO__PPC64EL=1
```

Comment:
Please add a FIXME comment here explaining why (if describing the mechanisms of LTO is too difficult - which is the case for me - just say that you ran into issues when building and attach a summary of the logs).

Reply:
Done. Another FIXME for package dependency `judy` has also been added.